### PR TITLE
Jessica Chan HW1 pull request 

### PR DESCRIPTION
**Pull request by chanjessica**
https://github.com/chanjessica/adopt-a-dog-spring2019

https://chanjessica.github.io/adopt-a-dog-spring2019/

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="zxx"> 
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (jimmynono) [inline blog.html]: Good to include that. 👍

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		<p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, conet labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea adqui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernamvoluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+		<p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, conet labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea adqui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernamvoluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+	 </div>
+     </div>  <!-- for flex-container-->
+ </section>
+
+
+<footer>
+	<img src="images/adoptadog-logo.svg" alt="dog LOGO">
+	<!--
```
Comment (jimmynono) [inline blog.html]: you can get rid of these comments. The img tags are loading the logo's just fine

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>A Dopt A Dog</title>
+	<link rel="stylesheet" type="text/css" href="styles/adopt-dog.css">
+  </head>
+  
+<body>
+<header>
```
Comment (jimmynono) [inline blog.html]: You'll want to indent everything in the body in by one level. A minor thing, but good practice.

File: contact.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+	<a href="index.html">Home</a> /
+	<a href="dogs.html">Dogs</a> / 
+	<a href="blog.html">Blog</a>
+</nav>
+
+  <form method="get" action="action_page.php">
+	<h3>Checkout</h3>
+    <label for="name">Name</label><br>
+    <input type="text" id="name" name="fullname">
+<p>
```
Comment (jimmynono) [inline contact.html]: Not sure what these extra 
p tags are

File: dogs.html [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+		<a href="index.html">Home</a> /
+		<a href="dogs.html">Dogs</a> / 
+		<a href="blog.html">Blog</a>
+	</nav>
+
+<section>
+	<h1>Hare are all of our Dogs! </h1>
+	<div class="flex-container">
+		<div>
+		<img src="images/murphy-card.jpg" alt="a boy huges a dog image">
```
Comment (jimmynono) [inline dogs.html]: You want to indent things in div tags

File: dogs.html [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+		<dl>
+		<dt>Murphy</dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+
+		<div>
+		<img src="images/poppy-card.jpg" alt="a boy huges a dog image">
+		<dl>
```
Comment (jimmynono) [inline dogs.html]: I don't think this is a good use of dt and dl tags. I'd consider using more basic tags.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

File: dogs.html [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+		<img src="images/luna-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Luna </dt>	
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+
+		<div>
+		<img src="images/stella-card.jpg" alt="a boy huges a dog image">
```
Comment (jimmynono) [inline dogs.html]: Minor thing, but all of your alt tags are the same, but use different images. Are the images the same?

**Comment by chanjessica** [comment]
HW2 add CSS

**Comment by chanjessica** [comment]
change <p> to </p>
add proportion font-size, padding

File: dogs.html [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+	</header>
+
+	<section>
+		<h1>Hare are all of our Dogs! </h1>
+
+		<div class="clear dogs-flex-container">
+			<div>
+				<img src="images/murphy-card.jpg" alt="murphy dog image">
+				<div class="dog">
+					<b>Murphy</b>
```
Comment (jimmynono) [inline dogs.html]: I'd recommend using strong instead of the b tag for bolds. You can set the style via css

**Comment by chanjessica** [comment]
reset the running total to 0 when navigate to a new page ( I used onunload) , that meant we can not carry the total from index page to the dogs page.  so all the selection has to make within the same page --that means you can't add buy 1 dog from the index page and 1 dog from the the dogs page.

**Comment by chanjessica** [comment]
change function to pass an array|object instead of passing 3 variables

**Comment by chanjessica** [comment]
add media queries

**Comment by chanjessica** [comment]
add background image and linear-gradient

File: blog.html [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+		<div class="head-checkout">
+			<div class="logo">
+				<figure>
+					<img src="images/adoptadog-logo.svg" alt="dog LOGO">
+					<figcaption>Seattle’s premiere dog adoption agency</figcaption>
+				</figure>
+			</div>
+			<div class="checkout">
+				<a href="#"><img src="images/cart-icon.svg" alt="checkout image"></a>$1234.56
+				<p>
```
Comment (jimmynono) [inline blog.html:28]: Empty p tag?

File: index.html [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+				<div class="dog">
+					<p>Jack</p>
+					<p><strong>Cost to Adopt:</strong> $100.30</p>
+					<p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+					<button onclick='buyIt(100.30)' type="button">Adopt</button>
+				</div>
+			</div>
+			
+			<div>
+				<img onclick='dogObj({name:"duffy",breed:"akita",cost:100.40})' src="images/duffy-card.jpg" alt="duffy dog image">
```
Comment (jimmynono) [inline index.html]: When using JS in HTML it's good to wrap the attribute in double quotes, and the things inside in single quotes

```
<img onclick="dogObj({name:'duffy', breed:'akita', cost:100.40})" src="images/duffy-card.jpg" alt="duffy dog image">
```

**Comment by chanjessica** [comment]
add jQuery

**Comment by chanjessica** [comment]
add language links to the web site  
change font-size on the checkout cart 
rework the header checkout button

**Pull request closed without merging**